### ISSUE TITLE
Letters in headings are to close to each other

### ISSUE THREAD

**Issue by hmosvoll** [PayEx/design.payex.com#11]
**Describe the bug**

The negative letter spacing cause some letters to have to spacing at all.

**To Reproduce**
- Go to [the Datepickers page](https://design.payex.com/docs/components/datepickers).
- Take a look at the h2 "Initial value"

**Possible fix**
Reduce negative margins and use em for dynamic spacing based on font size, such as `letter-spacing: -0.02em;`
Btw why does this only apply to h1 and h2?

**Screenshots**
![screen shot 2018-07-11 at 13 35 19](https://user-images.githubusercontent.com/16205454/42575038-077361f8-8520-11e8-8b4c-176b2bdd314e.png)

**Comment by erikhallaraaker** [comment]
I agree, on certain words/letter combos it makes it harder to read, maybe especially on h5's.
@stepandersen should probably give his 2 cents first though
